src/foam/nanos/crunch/ExpireUserCapabilityJunctionsCron.java

```diff
@@ -57,12 +60,21 @@ public void execute(X x) {
       if ( activeJunction.getIsInGracePeriod() || activeJunction.getGracePeriod() <= 0 ) {
         activeJunction.setStatus(CapabilityJunctionStatus.EXPIRED);
         activeJunction.setIsExpired(true);
+        userCapabilityJunctionDAO.put(activeJunction);
       } else {
         activeJunction.setIsInGracePeriod(true);
+
+        // update ucj data as renewable
+        FObject data = (FObject) activeJunction.getData();
```

Comment:
So here, if the activeJunction is not in gracePeriod, we will be updating the renewableData and activeJunction to be renewable ... why?